projects/ad719x_iio/app/app_config.c

```diff
@@ -55,10 +59,12 @@ static struct no_os_callback_desc ext_int_callback_desc = {
 
 /* I2C init parameters */
 static struct no_os_i2c_init_param no_os_i2c_init_params = {
-	.device_id = 0,
+	.device_id = I2C_DEVICE_ID,
 	.platform_ops = &i2c_ops,
 	.max_speed_hz = 100000,
+#if(ACTIVE_PLATFORM == MBED_PLATFORM)
```

Comment:
nit: Shall we define this for STM32 platform as well and remove this macro check similar to other projects?